architecture/core_objects/builds.adoc

```diff
@@ -302,19 +310,19 @@ Copy the value returned from the above command and place it into the
 ----
 ====
 
```

Comment:
Similar to the question in the Docker procedure above, I just want to clarify this is the correct procedure. We find the value using the `base64` command, then enter it into the secret.json file. Correct?